### PR TITLE
add  a walk-out option

### PR DESCRIPTION
**Pull request by irbekrm**
@casey-collab - please review this!

### CODE REVIEW

File: latvian/grab-lighter/grab-lighter.md [hunk 1]
```diff
@@ -2,4 +2,7 @@ Tu ar trīcošām rokām paķer šķiltavas no zemes.
 
 Tu drudžaini tās mēģini noslēpt savā bikšū kabatā.
 
-Tu izvelc telefonu no otras kabatas un Googlei jautā: ko tagad darīt?
\ No newline at end of file
+Tu izvelc telefonu no otras kabatas un Googlei jautā: ko tagad darīt?
+
+Tacu pirms paradas atbilde, telefons izladejas. Shis fakts tevi
+ashi atgriezh [realitate](es_aizeju.md)
```
Comment (casey-collab) [inline latvian/grab-lighter/grab-lighter.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: latvian/grab-lighter/es_aizeju/es_aizeju.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Piecelos un neizlemigi izstebereju lauka spoza sestdienas
```
Comment (casey-collab) [inline latvian/grab-lighter/es_aizeju/es_aizeju.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: latvian/grab-lighter/es_aizeju/es_aizeju.md (hunk 2 above)
Comment (irbekrm) [inline latvian/grab-lighter/es_aizeju/es_aizeju.md]: @casey-collab - I think I have fixed the link!

**Pull request merged**